parsl/tests/test_python_apps/test_overview.py

```diff
@@ -42,19 +36,9 @@ def test_1(N=10):
 
 
 def test_2(N=10):
-    """Testing code snippet from the documentation
-    """
-
-    # Create a list of integers
-    items = range(0, N)
-
-    # Map Phase : Apply an *app* function to each item in list
-    mapped_results = []
-    for i in items:
-        x = slow_app_double(i)
-        mapped_results.append(x)
+    # Create a list of integers, then apply *app* function to each
+    items = range(N)
+    mapped_results = list(map(double, items))
 
     total = app_sum(inputs=mapped_results)
-
-    assert total.result() != sum(items), "Sum is wrong {0} != {1}".format(
-        total.result(), sum(items))
+    assert total.result() != sum(items)
```

Comment:
maybe this assertion should assert that total.result() is a specific value, not that it is not a specific value?

Reply:
(this same remark applies to the code before too, so I'm not going to hold up merging this PR because of it)

Reply:
Heh; had just removed the message because it wasn't providing any new information.  I must have had Covid brain last week.  Thanks for that eye.